### ISSUE TITLE
SyntaxError: Named export 'falseFunc' not found. 

### ISSUE THREAD

**Issue by ooip** [fb55/nth-check#208]
nth-check is a dependency of cheerio. I just updated all my packages and now I'm receiving the below error:

```javascript
file:///path/to/my-module/node_modules/nth-check/lib/esm/compile.js:1
import { trueFunc, falseFunc } from "boolbase";
                   ^^^^^^^^^
SyntaxError: Named export 'falseFunc' not found. The requested module 'boolbase' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from 'boolbase';
const { trueFunc, falseFunc } = pkg;

    at ModuleJob._instantiate (node:internal/modules/esm/module_job:127:21)
    at async ModuleJob.run (node:internal/modules/esm/module_job:191:5)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:337:24)
    at async loadESM (node:internal/process/esm_loader:88:5)
    at async handleMainPromise (node:internal/modules/run_main:61:12)
```

**Comment by jaywcjlove** [comment]
@fb55 Big changes should be released in version 3.0.

**Comment by jaywcjlove** [comment]
rollback dependency nth-check v2.1.0 to 2.0.1
@1zilc @ooip

**Comment by fb55** [comment]
Fixed in https://github.com/fb55/nth-check/commit/aeeb0676f6af940e8639f64e27781443694ca05e, released as `2.1.1`.
